src/main/java/org/jenkinsci/plugins/github/pullrequest/GitHubPRTrigger.java

```diff
@@ -66,6 +104,7 @@
  */
 public class GitHubPRTrigger extends Trigger<AbstractProject<?, ?>> {
     private static final Logger LOGGER = LoggerFactory.getLogger(GitHubPRTrigger.class);
+    private static final Cause NO_CAUSE = null;
```

Comment:
this can be redundant because null required for scheduleBuild2 just because scheduleBuild2 has many methods and maybe there is more suitable method.